### PR TITLE
docs: updated development info for contributors

### PR DESCRIPTION
**Pull request by abarmina**
updated information in the sections "Pages" and "Scripts"

### CODE REVIEW

File: docs/contribute/pages.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Development: Pages
 
-Demo site built with [Eleventy](https://www.11ty.dev/docs/) - a simple static site generator. 
+The demo site is built with [Eleventy](https://www.11ty.dev/docs/) - a simple static site generator. 
 
-[Nunjucks](https://mozilla.github.io/nunjucks/getting-started.htmlх) was picked as a template language. 
+[Nunjucks](https://mozilla.github.io/nunjucks/) and [Markdown](https://www.markdownguide.org/) are picked as template languages.
 
-Project deploys to GitHub pages.
+The project is deployed using a GitHub workflow.
```
Comment (ala-n) [inline docs/contribute/pages.md:7]: Supercosmetic: `The project is deployed to GitHub Pages using a GitHub workflow.`

File: docs/contribute/scripts.md [hunk 2]
```diff
@@ -5,12 +5,11 @@
 Here is a list of available npm scripts for local development:
 
 - `npm start` or `npm run start` - start demo server locally. Runs local build, watch and browsersync.  
-  Uses `:3001` port (BrowserSync) and `:3002` port (origin)
+  Uses `:3005` port
```
Comment (ala-n) [inline docs/contribute/scripts.md:8]: 👍🏻

**Review by ala-n — APPROVED** [review]

**Review by dshovchko — APPROVED** [review]

File: docs/contribute/pages.md (hunk 1 above)
Comment (abarmina) [reply docs/contribute/pages.md:7]: ah, at first I thought to write like this

**Pull request merged**